### PR TITLE
ranger: new package at 1.9.3

### PR DESCRIPTION
**Pull request by aicsx**
## Description of package
ranger is a console file manager with VI key bindings. It provides a minimalistic and nice curses interface with a view on the directory hierarchy. It ships with rifle, a file launcher that is good at automatically finding out which program to use for what file type.

## Installed manifest of package

(`cat /var/db/kiss/installed/ranger/manifest`)

```
/var/db/kiss/installed/ranger/version
/var/db/kiss/installed/ranger/sources
/var/db/kiss/installed/ranger/patches/081e73152a9391211770fab56676d7d974413ae6.patch
/var/db/kiss/installed/ranger/patches/
/var/db/kiss/installed/ranger/manifest
/var/db/kiss/installed/ranger/depends
/var/db/kiss/installed/ranger/checksums
/var/db/kiss/installed/ranger/build
/var/db/kiss/installed/ranger/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/rifle.1
/usr/share/man/man1/ranger.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/doc/ranger/tools/print_keys.py
/usr/share/doc/ranger/tools/print_colors.py
/usr/share/doc/ranger/tools/performance_test.py
/usr/share/doc/ranger/tools/convert_papermode_to_metadata.py
/usr/share/doc/ranger/tools/
/usr/share/doc/ranger/examples/vim_file_chooser.vim
/usr/share/doc/ranger/examples/shell_subshell_notice.sh
/usr/share/doc/ranger/examples/shell_automatic_cd.sh
/usr/share/doc/ranger/examples/rifle_sxiv.sh
/usr/share/doc/ranger/examples/rifle_different_file_opener.conf
/usr/share/doc/ranger/examples/rc_emacs.conf
/usr/share/doc/ranger/examples/plugin_pmount_dynamic.py
/usr/share/doc/ranger/examples/plugin_pmount.py
/usr/share/doc/ranger/examples/plugin_new_sorting_method.py
/usr/share/doc/ranger/examples/plugin_new_macro.py
/usr/share/doc/ranger/examples/plugin_linemode.py
/usr/share/doc/ranger/examples/plugin_ipc.py
/usr/share/doc/ranger/examples/plugin_hello_world.py
/usr/share/doc/ranger/examples/plugin_file_filter.py
/usr/share/doc/ranger/examples/plugin_fasd_add.py
/usr/share/doc/ranger/examples/plugin_chmod_keybindings.py
/usr/share/doc/ranger/examples/plugin_avfs.py
/usr/share/doc/ranger/examples/fish_automatic_cd.fish
/usr/share/doc/ranger/examples/README
/usr/share/doc/ranger/examples/
/usr/share/doc/ranger/config/scope.sh
/usr/share/doc/ranger/config/rifle.conf
/usr/share/doc/ranger/config/rc.conf
/usr/share/doc/ranger/config/commands.py
/usr/share/doc/ranger/config/colorschemes/snow.py
/usr/share/doc/ranger/config/colorschemes/jungle.py
/usr/share/doc/ranger/config/colorschemes/default.py
/usr/share/doc/ranger/config/colorschemes/
/usr/share/doc/ranger/config/
/usr/share/doc/ranger/colorschemes.md
/usr/share/doc/ranger/README.md
/usr/share/doc/ranger/HACKING.md
/usr/share/doc/ranger/CHANGELOG.md
/usr/share/doc/ranger/
/usr/share/doc/
/usr/share/applications/ranger.desktop
/usr/share/applications/
/usr/share/
/usr/lib/python3.8/site-packages/ranger_fm-1.9.3-py3.8.egg-info
/usr/lib/python3.8/site-packages/ranger/gui/widgets/view_multipane.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/view_miller.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/view_base.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/titlebar.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/taskview.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/statusbar.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/pager.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/console.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/browsercolumn.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/view_multipane.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/view_multipane.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/view_miller.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/view_miller.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/view_base.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/view_base.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/titlebar.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/titlebar.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/taskview.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/taskview.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/statusbar.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/statusbar.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/pager.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/pager.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/console.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/console.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/browsercolumn.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/browsercolumn.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__pycache__/
/usr/lib/python3.8/site-packages/ranger/gui/widgets/__init__.py
/usr/lib/python3.8/site-packages/ranger/gui/widgets/
/usr/lib/python3.8/site-packages/ranger/gui/ui.py
/usr/lib/python3.8/site-packages/ranger/gui/mouse_event.py
/usr/lib/python3.8/site-packages/ranger/gui/displayable.py
/usr/lib/python3.8/site-packages/ranger/gui/curses_shortcuts.py
/usr/lib/python3.8/site-packages/ranger/gui/context.py
/usr/lib/python3.8/site-packages/ranger/gui/colorscheme.py
/usr/lib/python3.8/site-packages/ranger/gui/color.py
/usr/lib/python3.8/site-packages/ranger/gui/bar.py
/usr/lib/python3.8/site-packages/ranger/gui/ansi.py
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/ui.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/ui.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/mouse_event.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/mouse_event.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/displayable.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/displayable.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/curses_shortcuts.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/curses_shortcuts.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/context.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/context.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/colorscheme.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/colorscheme.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/color.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/color.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/bar.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/bar.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/ansi.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/ansi.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/gui/__pycache__/
/usr/lib/python3.8/site-packages/ranger/gui/__init__.py
/usr/lib/python3.8/site-packages/ranger/gui/
/usr/lib/python3.8/site-packages/ranger/ext/widestring.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/vcs.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/svn.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/hg.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/git.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/bzr.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/vcs.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/vcs.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/svn.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/svn.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/hg.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/hg.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/git.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/git.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/bzr.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/bzr.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__pycache__/
/usr/lib/python3.8/site-packages/ranger/ext/vcs/__init__.py
/usr/lib/python3.8/site-packages/ranger/ext/vcs/
/usr/lib/python3.8/site-packages/ranger/ext/spawn.py
/usr/lib/python3.8/site-packages/ranger/ext/signals.py
/usr/lib/python3.8/site-packages/ranger/ext/shutil_generatorized.py
/usr/lib/python3.8/site-packages/ranger/ext/shell_escape.py
/usr/lib/python3.8/site-packages/ranger/ext/safe_path.py
/usr/lib/python3.8/site-packages/ranger/ext/rifle.py
/usr/lib/python3.8/site-packages/ranger/ext/relative_symlink.py
/usr/lib/python3.8/site-packages/ranger/ext/popen_forked.py
/usr/lib/python3.8/site-packages/ranger/ext/openstruct.py
/usr/lib/python3.8/site-packages/ranger/ext/next_available_filename.py
/usr/lib/python3.8/site-packages/ranger/ext/mount_path.py
/usr/lib/python3.8/site-packages/ranger/ext/logutils.py
/usr/lib/python3.8/site-packages/ranger/ext/lazy_property.py
/usr/lib/python3.8/site-packages/ranger/ext/keybinding_parser.py
/usr/lib/python3.8/site-packages/ranger/ext/iter_tools.py
/usr/lib/python3.8/site-packages/ranger/ext/img_display.py
/usr/lib/python3.8/site-packages/ranger/ext/human_readable.py
/usr/lib/python3.8/site-packages/ranger/ext/hash.py
/usr/lib/python3.8/site-packages/ranger/ext/get_executables.py
/usr/lib/python3.8/site-packages/ranger/ext/direction.py
/usr/lib/python3.8/site-packages/ranger/ext/curses_interrupt_handler.py
/usr/lib/python3.8/site-packages/ranger/ext/cached_function.py
/usr/lib/python3.8/site-packages/ranger/ext/accumulator.py
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/widestring.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/widestring.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/spawn.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/spawn.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/signals.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/signals.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/shutil_generatorized.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/shutil_generatorized.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/shell_escape.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/shell_escape.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/safe_path.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/safe_path.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/rifle.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/rifle.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/relative_symlink.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/relative_symlink.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/popen_forked.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/popen_forked.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/openstruct.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/openstruct.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/next_available_filename.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/next_available_filename.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/mount_path.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/mount_path.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/logutils.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/logutils.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/lazy_property.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/lazy_property.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/keybinding_parser.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/keybinding_parser.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/iter_tools.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/iter_tools.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/img_display.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/img_display.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/human_readable.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/human_readable.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/hash.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/hash.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/get_executables.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/get_executables.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/direction.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/direction.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/curses_interrupt_handler.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/curses_interrupt_handler.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/cached_function.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/cached_function.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/accumulator.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/accumulator.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/ext/__pycache__/
/usr/lib/python3.8/site-packages/ranger/ext/__init__.py
/usr/lib/python3.8/site-packages/ranger/ext/
/usr/lib/python3.8/site-packages/ranger/data/scope.sh
/usr/lib/python3.8/site-packages/ranger/data/mime.types
/usr/lib/python3.8/site-packages/ranger/data/
/usr/lib/python3.8/site-packages/ranger/core/tab.py
/usr/lib/python3.8/site-packages/ranger/core/shared.py
/usr/lib/python3.8/site-packages/ranger/core/runner.py
/usr/lib/python3.8/site-packages/ranger/core/metadata.py
/usr/lib/python3.8/site-packages/ranger/core/main.py
/usr/lib/python3.8/site-packages/ranger/core/loader.py
/usr/lib/python3.8/site-packages/ranger/core/linemode.py
/usr/lib/python3.8/site-packages/ranger/core/fm.py
/usr/lib/python3.8/site-packages/ranger/core/filter_stack.py
/usr/lib/python3.8/site-packages/ranger/core/actions.py
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/tab.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/tab.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/shared.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/shared.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/runner.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/runner.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/metadata.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/metadata.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/main.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/main.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/loader.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/loader.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/linemode.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/linemode.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/fm.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/fm.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/filter_stack.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/filter_stack.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/actions.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/actions.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/core/__pycache__/
/usr/lib/python3.8/site-packages/ranger/core/__init__.py
/usr/lib/python3.8/site-packages/ranger/core/
/usr/lib/python3.8/site-packages/ranger/container/tags.py
/usr/lib/python3.8/site-packages/ranger/container/settings.py
/usr/lib/python3.8/site-packages/ranger/container/history.py
/usr/lib/python3.8/site-packages/ranger/container/fsobject.py
/usr/lib/python3.8/site-packages/ranger/container/file.py
/usr/lib/python3.8/site-packages/ranger/container/directory.py
/usr/lib/python3.8/site-packages/ranger/container/bookmarks.py
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/tags.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/tags.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/settings.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/settings.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/history.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/history.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/fsobject.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/fsobject.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/file.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/file.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/directory.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/directory.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/bookmarks.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/bookmarks.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/container/__pycache__/
/usr/lib/python3.8/site-packages/ranger/container/__init__.py
/usr/lib/python3.8/site-packages/ranger/container/
/usr/lib/python3.8/site-packages/ranger/config/rifle.conf
/usr/lib/python3.8/site-packages/ranger/config/rc.conf
/usr/lib/python3.8/site-packages/ranger/config/commands_sample.py
/usr/lib/python3.8/site-packages/ranger/config/commands.py
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/commands_sample.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/commands_sample.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/commands.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/commands.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/config/__pycache__/
/usr/lib/python3.8/site-packages/ranger/config/__init__.py
/usr/lib/python3.8/site-packages/ranger/config/
/usr/lib/python3.8/site-packages/ranger/colorschemes/solarized.py
/usr/lib/python3.8/site-packages/ranger/colorschemes/snow.py
/usr/lib/python3.8/site-packages/ranger/colorschemes/jungle.py
/usr/lib/python3.8/site-packages/ranger/colorschemes/default.py
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/solarized.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/solarized.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/snow.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/snow.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/jungle.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/jungle.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/default.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/default.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/colorschemes/__pycache__/
/usr/lib/python3.8/site-packages/ranger/colorschemes/__init__.py
/usr/lib/python3.8/site-packages/ranger/colorschemes/
/usr/lib/python3.8/site-packages/ranger/api/commands.py
/usr/lib/python3.8/site-packages/ranger/api/__pycache__/commands.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/api/__pycache__/commands.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/api/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/api/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/api/__pycache__/
/usr/lib/python3.8/site-packages/ranger/api/__init__.py
/usr/lib/python3.8/site-packages/ranger/api/
/usr/lib/python3.8/site-packages/ranger/__pycache__/__init__.cpython-38.pyc
/usr/lib/python3.8/site-packages/ranger/__pycache__/__init__.cpython-38.opt-1.pyc
/usr/lib/python3.8/site-packages/ranger/__pycache__/
/usr/lib/python3.8/site-packages/ranger/__init__.py
/usr/lib/python3.8/site-packages/ranger/
/usr/lib/python3.8/site-packages/
/usr/lib/python3.8/
/usr/lib/
/usr/bin/rifle
/usr/bin/ranger
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/ranger/depends [hunk 1]
```diff
@@ -0,0 +1,3 @@
+mandoc
```
Comment (dylanaraps) [inline community/ranger/depends:1]: Is `mandoc` really a dependency? ie, it won't build without it?

**Comment by aicsx** [comment]
Yes for now It is, read https://github.com/ranger/ranger/issues/1892

Whithout mandoc ranger crash, It was 1st build. But it has been reported and will be corrected in the next release

File: community/ranger/build [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/ranger/build:3]: `make DESTDIR="$1" PREFIX=/usr install

**Comment by aicsx** [comment]
wait wait... problem ...

**Comment by aicsx** [comment]
Ok this error occurs in the variable case TERM=screen256-color ... in all other cases, that is: TERM = xterm or similar there are no problems, the fix will be released in the next release 1.9.4

let me know if I have to release it as it is now or if you want to add the fix in post install or if I have to wait for the official release of 1.9.4

**Comment by aicsx** [comment]
patch apply. now work.

**Pull request merged**